site/app/views/forum/ForumThreadView.php

```diff
@@ -1138,31 +1138,14 @@ public function createPost(array $thread, array $post, $unviewed_posts, $first,
         $post_user_info = [];
 
         $merged_thread = $is_merged_thread && $userAccessFullGrading;
-        if ($userAccessFullGrading) {
-            $info_name = $given_name . " " . $family_name . " (" . $post['author_user_id'] . ")";
-            $visible_user_json = json_encode($visible_username);
-            $pronouns = trim($author_info["pronouns"]);
-            $display_pronouns = $author_info["display_pronouns"];
-            $info_name = json_encode($info_name);
-            $jscriptAnonFix = $post['anonymous'] ? 'true' : 'false';
-            $jscriptAnonFix = json_encode($jscriptAnonFix);
-
-            $post_user_info = [
-                "info_name" => $info_name,
-                "visible_user_json" => $visible_user_json,
-                "jscriptAnonFix" => $jscriptAnonFix,
-                "pronouns" => $pronouns,
-                "display_pronouns" => $display_pronouns
-            ];
-        }
```

Comment:
For other reviewers: this code is reused in a separate if block below, making it redundant here.